spyder_kernels/utils/pythonenv.py

```diff
@@ -61,6 +62,35 @@ def get_conda_env_path(pyexec, quote=False):
     return conda_env
 
 
+def get_pixi_manifest_path_and_env_name(pyexec, quote=False):
```

Comment:
Let's use `pathlib` instead of `os.path` for the path manipulations in this function because the logic is easier to follow.